README.md

```diff
@@ -1,70 +1,137 @@
+automated upgrades are enabled.
 
-```sh
-podman run --rm -ti \
-    -v /tmp/cvo:/tmp/cvo:z \
-    <release image> \
-        render \
-        --output-dir=/tmp/cvo \
-        --release-image="<release image>"
-```
+When `spec.desiredUpdate` does not match the current cluster version, CVO will
+start updating the cluster. It downloads the release payload image, validates
+it, and [systematically reconciles](https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/user/reconciliation.md)
+the Cluster Operator resources to match the updated manifests delivered
+in the release payload image.
 
-`<release image>` can be personal release image generated using [this](#building-release-image-using-local-cvo) or Origin's release image like `registry.ci.openshift.org/openshift/origin-release:v4.0`.
+## Troubleshooting
 
-## Running CVO tests
+A typical OpenShift cluster will have a `Deployment` resource called
+`cluster-version-operator` in the `openshift-cluster-version` namespace,
+configured to run a single CVO replica. Confirm that its `Pod` is up and optionally
+inspect its log:
 
-```sh
-# Run all unit tests
-go test ./...
+```console
+$ oc get deployment -n openshift-cluster-version cluster-version-operator
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+cluster-version-operator   1/1     1            1           2y227d
 
-# Run integration tests against a cluster (creates content in a given namespace)
-# Requires the CVO CRD to be installed.
-export KUBECONFIG=<admin kubeconfig>
-TEST_INTEGRATION=1 go test ./... -test.run=^TestIntegration
+$ oc get pods -n openshift-cluster-version -l k8s-app=cluster-version-operator
+NAME                                        READY   STATUS    RESTARTS   AGE
+cluster-version-operator-7fdf58f594-ct92x   1/1     Running   1          85d
+
+$ oc logs -n openshift-cluster-version -l k8s-app=cluster-version-operator
+...
 ```
 
-## Contributing
+The CVO follows the [Kubernetes API conventions](https://github.com/kubernetes/community/blob/4c9ef2d/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) and sets `Conditions` in the status of its `ClusterVersion` resource:
+
+```console
+$ oc get clusterversion version -o jsonpath='{.status.conditions}' | jq
+[
+  {
+    "lastTransitionTime": "2020-04-16T19:39:45Z",
+    "message": "Done applying 4.11.3",
+    "status": "True",
+    "type": "Available"
+  },
+  {
+    "lastTransitionTime": "2022-11-10T23:40:34Z",
+    "status": "False",
+    "type": "Failing"
+  },
+  {
+    "lastTransitionTime": "2022-09-06T06:58:04Z",
+    "message": "Cluster version is 4.11.3",
+    "status": "False",
+    "type": "Progressing"
+  },
+  ...
+```
 
-[Contributions welcome](CONTRIBUTING.md)!
+### Troubleshooting Cluster Upgrades
 
-## Documentation
+TODO: Are there good links for this? Inspecting update pod? Inspecting `ClusterOperator` resources?
```

Comment:
Hopefully our existing condition strings are already actionable enough.  If they aren't, we should improve those strings, or the context `oc` and the web-console add to those strings, instead of trying to paper over any gaps in our GitHub README.

Reply:
Yeah, I think this is "troubleshooting upgrades", not "troubleshooting" CVO, so I'd like this section to be super short and ideally just lead elsewhere :+1: